src/Cake.Common/Tools/NUnit/NUnit3Settings.cs

```diff
@@ -246,6 +246,12 @@ public sealed class NUnit3Settings : ToolSettings
         /// </value>
         public int? Agents { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating the path to an alternative app.config file to load.
+        /// </summary>
+        /// <value>The location that NUnit should load an alternative app.config file from.</value>
+        public FilePath ConfigurationFile { get; set; }
```

Comment:
I was a little too quick approving this PR. Could we make this argument match the NUnit argument name and rename it to `ConfigFile`?

Reply:
Sure thing.